cg/cli/base.py

```diff
@@ -117,3 +118,4 @@ def init(context: CGConfig, reset: bool, force: bool):
 base.add_command(generate_cmd)
 base.add_command(downsample)
 base.add_command(validate)
+base.add_command(sequencing_qc)
```

Comment:
Do you think this would fit under the "validate" command?

`cg validate sequencing-qc`?

Reply:
I'd rather keep the sequencing quality control as a separate top level command for now